lib/read-body.js

```diff
@@ -81,6 +82,10 @@ const collectFiles = (busboy, limits, fileFilter) => new Promise((resolve, rejec
       return reject(err)
     }
 
+    if (!createWriteStream) {
+      createWriteStream = (await import('fs-temp')).createWriteStream
+    }
```

Comment:
You don't need to perform manual caching; Node.js handles it on its own.
```suggestion
    const {createWriteStream} = await import('fs-temp');
```

Reply:
not sure why but without caching tests start to fail with 
`Timeout of 2000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.`

Reply:
This is hardly possible, could you push these changes to see which version and which test fails in the pipeline?

Reply:
pushed, seems like it fails on cases when array of files are uploaded